<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationPage</class>
 <widget class="QMainWindow" name="RegistrationPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Registrazione</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="fotoprofilo">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="images.qrc">:/images/img/logo.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="username">
             <property name="text">
              <string>Username</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="usernameEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="nome">
             <property name="text">
              <string>Nome e cognome</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="nomeEdit"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="email">
             <property name="text">
              <string>Email</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="emailEdit"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="password">
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="passwordEdit"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="password2">
             <property name="text">
              <string>Inserire nuovamente la password</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="password2Edit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>50</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QPushButton" name="registrati">
           <property name="text">
            <string>Registrati</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="annulla">
           <property name="text">
            <string>Annulla</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>usernameEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>registrati</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nomeEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>registrati</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>emailEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>registrati</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>passwordEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>registrati</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>password2Edit</sender>
   <signal>returnPressed()</signal>
   <receiver>registrati</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
